README.md

```diff
@@ -24,23 +24,23 @@ to think about your problems that involves a weave between natural language task
 and traditional computing tasks which deterministic Python code gives you.
 
 Neuro-San also offers:
-    * channels for private data (aka sly_data) that should be kept out of LLM chat streams
-    * LLM-provider agnosticism and extensibiltiy of data-only-configured LLMs when new hotness arrives.
-    * agent-specific LLM specifications - use the right LLM for the cost/latency/context-window/data-privacy each agent needs.
-    * fallback LLM specifications for when your fave goes down.
-    * powerful debugging information for gaining insight into your mutli-agent systems.
-    * server-readiness at scale
-    * enabling of distributed agent webs that call each other to work together, wheverer they are hosted.
-    * security-by-default - you set what private data is to be shared downstream/upstream
+
+* channels for private data (aka sly_data) that should be kept out of LLM chat streams
+* LLM-provider agnosticism and extensibiltiy of data-only-configured LLMs when new hotness arrives.
+* agent-specific LLM specifications - use the right LLM for the cost/latency/context-window/data-privacy each agent needs.
+* fallback LLM specifications for when your fave goes down.
+* powerful debugging information for gaining insight into your mutli-agent systems.
+* server-readiness at scale
+* enabling of distributed agent webs that call each other to work together, wheverer they are hosted.
+* security-by-default - you set what private data is to be shared downstream/upstream
```

Comment:
Fix indenting.  This and other fixes below were found and fixed with markdownlint-cli2
Thanks to @donn-leaf for recommendation.

Reply:
Only linter item I did not fix was line length.